src/cli/subcommands/snapshot_cmd.rs

```diff
@@ -94,6 +99,33 @@ pub enum SnapshotCommands {
         #[arg(long, default_value_t = false)]
         force: bool,
     },
+    /// Filecoin keeps track of "the state of the world", including:
+    /// wallets and their balances;
+    /// storage providers and their deals;
+    /// etc...
+    ///
+    /// It does this by (essentially) hashing the state of the world.
+    ///
+    /// The world can change when new blocks are mined and transmitted.
+    /// A block may contain a message to e.g transfer FIL between two parties.
+    /// Blocks are ordered by "epoch", which can be thought of as a timestamp.
+    ///
+    /// Snapshots contain (among other things) these messages.
+    ///
+    /// The command calculates the state of the world at EPOCH-1, applies all
+    /// the messages at EPOCH, and prints the resulting hash of the state of the world.
```

Comment:
This implied that _two_ states were computed, rather than just one.

```suggestion
    /// The command takes the state of the world at EPOCH-1, applies all
    /// the messages at EPOCH, and prints the resulting hash of the state of the world.
```

Reply:
Surely two states are computed? we need the state root for n-1 to calculate the state at n? or am I mistaken